pkg/dir.traversal.targets

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="..\dir.traversal.targets" />
+  
+  <PropertyGroup Condition="'$(OfficialBuildId)' != ''">
+    <!-- During an official build, only build identity packages in the AllConfigurations build -->
+    <SkipBuildIdentityPackage Condition="'$(BuildAllConfigurations)' != 'true'">true</SkipBuildIdentityPackage>
```

Comment:
I don't think the "BuildAllConfigurations" concept applies to core-setup.  perhaps we just pick one of the builds there and pass an extra flag signifying that it should build the identity package.  Though, that would require disabling running tests at the same time for those builds, so maybe just build identity packages everywhere but apply the flag to the publishing step to prevent trying to publish multiple identity packages.